# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: devtools/shared/specs/index.js
# Commit: 895cf08432c4
# Full Hash: 895cf08432c40fe371ff83e846cb161430c6be4b
# Author: Nicolas Chevobbe <nchevobbe@mozilla.com>
# Date: 2020-10-27 04:41:26
# Regressor Bug: 1633712
# File Overlap Count: 1
# Description:
#   Bug 1633712 - [devtools] Create WorkerTargets as soon as possible. r=ochameau,devtools-backward-compat-reviewers.
#   
#   This patch adds support for dedicated worker targets in the Watcher actor.
#   Shared and Service workers are not handled yet.
#   
# ==============================================================================

diff -r 2ce79a9ed2d5 -r 895cf08432c4 devtools/shared/specs/index.js
--- a/devtools/shared/specs/index.js	Mon Oct 26 12:52:26 2020 +0000
+++ b/devtools/shared/specs/index.js	Mon Oct 26 12:53:43 2020 +0000
@@ -290,7 +290,7 @@
   {
     types: ["workerTarget"],
     spec: "devtools/shared/specs/targets/worker",
-    front: null,
+    front: "devtools/client/fronts/targets/worker",
   },
   {
     types: ["thread"],